CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 * Fix `where` chaining on `ActiveQuery` (https://github.com/Beyond-Finance/active_force/pull/7)
 * Add `find_by!` which raises `ActiveForce::RecordNotFound` if nothing is found. (https://github.com/Beyond-Finance/active_force/pull/8)
 
+## 0.10.1
```

Comment:
Let's get this in with the other `0.10.0` stuff.

Reply:
Done.